# DetectedProduct message
# This structure contains the information of a detected product.

string briefcase_id
# Product type
string product_type

# Whether or not the product is faulty
string product_name

# Target pose
geometry_msgs/Pose product_pose
